scripts/jenkins/build

```diff
@@ -52,6 +55,31 @@ fi
 
 ./autogen.sh |& tee $ARTIFACTS/configure.log
 
+if [[ $TRANSPORT == "lnet" ]]; then
+	PACKAGE='lustre-client'
+else
+	PACKAGE='libfabric'
+fi
+
+# Checking if package is available or not
+if [[ ! -z $(rpm -qa $PACKAGE) ]]; then
+        echo "$PACKAGE package is present"
+else
+        echo "need to install $PACKAGE module"
+	yum install ${PACKAGE}* -y
+fi
+
+if [[ "$PACKAGE" == "libfabric" ]]; then
+        echo "You are using default $PACKAGE module"
+else
+	if  [[ ! -z $(rpm -qa | grep libfabric) ]]; then
```

Comment:
@upendrapatwardhan : Can you add a todo here for selection of transport based on configure option "--with-default-xprt=LF|SOCK|LNET" and which will update  xprt.in.
Then value of this transport will be used by UT's and ST's.

Reply:
@nikhilpatil2995 
Could you add the above TODO as I am not able to push commits to your repo.